article.html

```diff
@@ -7,37 +7,76 @@
 
 	<body>
 		<header>
-			<p>This will be the nav</p>
+			<a href="index.html" class="logo">Skinclusive</a>
+			<nav name="top">
+				<ul>
+					<li><a href="quiz.html">skin type quiz</a></li>
+					<li class="dropdown"><a href="all-products.html">products &#9662;</a>
+						<div class="dropdown-content">
+							<a href="#">oily skin products</a>
+							<a href="#">dry skin products</a>
+							<a href="#">combo skin products</a>
+							<a href="#">sensitive skin products</a>
+						</div>
+					</li>
+					<li class="dropdown"><a href="article-list.html">articles &#9662;</a>
+						<div class="dropdown-content">
+							<a href="#">oily skin articles</a>
+							<a href="#">dry skin articles</a>
+							<a href="#">combo skin articles</a>
+							<a href="#">sensitive skin articles</a>
+						</div>
+					</li>
+				</ul>
+			</nav>
 		</header>
 		
 		<div class="wrapper">
             <!-- This div is for the template for all articles -->
             <div class="article">
                 <h1>Article header here</h1>
-                <h2>Author and publish date here</h2>
+                <h2>Skinclusive | DD/MM/YYYY</h2>
 
                 <div class="video">
                     <p>Placeholder for a video</p>
                 </div>
-                <p>Article text will go here. Relevant images will also be spread out through this page</p>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor 

```

Comment:
In my final layout, I added dummy text to give me a clearer idea of how text and images will sit on the page rather than just having a blank box with only 2 sentences.